src/app/legacy/containers/ATIAnalytics/params/index.js

```diff
@@ -1,3 +1,19 @@
+import {
+  HOME_PAGE,
+  STORY_PAGE,
+  MEDIA_ASSET_PAGE,
+  CORRESPONDENT_STORY_PAGE,
+  ARTICLE_PAGE,
+  FRONT_PAGE,
+  TOPIC_PAGE,
+  MEDIA_ARTICLE_PAGE,
+  FEATURE_INDEX_PAGE,
+  MOST_READ_PAGE,
+  MOST_WATCHED_PAGE,
+  INDEX_PAGE,
+  PHOTO_GALLERY_PAGE,
+  MEDIA_PAGE,
+} from '#app/routes/utils/pageTypes';
```

Comment:
Unrelated to this work but it would be good to type-check these at some stage, similar to what we've done in the BFF.